piff/decamutil.py

```diff
@@ -0,0 +1,214 @@
+#
+# $Rev::                                                              $:
+# $Author::                                                           $:
+# $LastChangedDate::                                                  $:
+#
+# Utility methods for DECam
+#
+
+import numpy as np
+
+class decaminfo(object):
+    """ decaminfo is a class used to contain DECam geometry information and various utility routines
+    """
+
+    def info(self):
+        # info returns a dictionary chock full of info on the DECam geometry
+        # keyed by the CCD name
+
+        infoDict = {}

```

Comment:
Slightly neater here would be to have infoDict be a class attribute that gets set only once if it is not defined yet:
```
class DECamInfo(object):
    _infoDict = None

    @property
    def infoDict(self):
        if DECamInfo._infoDict is None:
            DECamInfo._infoDict = {
                'S1' : {"xCenter":  -16.908,"yCenter":-191.670, "FAflag":False, "CCDNUM":25},
                'S2' : {"xCenter":  -16.908,"yCenter":-127.780, "FAflag":False, "CCDNUM":26},
                ...
            }
        return DECamInfo._infoDict
```